python_modules/libraries/dagster-dg/dagster_dg_tests/test_cache.py

```diff
@@ -10,8 +10,8 @@
 
 # For all cache tests, avoid setting up venv in example code location so we do not prepopulate the
 # cache (which is part of the venv setup routine).
-example_code_location = partial(isolated_example_code_location_foo_bar, skip_venv=True)
-cache_runner_args = {"verbose": True, "require_local_venv": False}
+example_code_location = partial(isolated_example_code_location_foo_bar, populate_cache=False)
```

Comment:
this is the default anyways, right?

Reply:
good catch, the default should actually be `True` so I will change